### ISSUE TITLE
part_b_job1.py job submission potential typo

### ISSUE THREAD

**Issue by unmovingcastle** [osu-particle-astrophysics/GENETIS_HPol#11]
should probably be `--array=1-{npop}...` instead.

**Comment by ezio-melotti** [comment]
Here?
https://github.com/osu-particle-astrophysics/GENETIS_HPol/blob/954abc23624a38a6eaf0069ff6595cbf88524d10/Evolutionary_loop/Loop_Parts/Part_B/part_b_job1.py#L186

**Comment by unmovingcastle** [comment]
Yes, I noticed it but didn't want to change it just yet. I'll fix it in the future.
In the meantime, should I go ahead and merge the jbook branch (PR #10) since the _build directory issue is sorted out.
